edx/edx-ora2
# Fix file upload to support non-ASCII file names

**TL;DR -** allow non-ASCII file names in ORA uploads.

JIRA: [EDUCATOR-5566](https://openedx.atlassian.net/browse/EDUCATOR-5566)

**What changed?**

- Modify upload headers to encode the filename, allowing for non-ASCII file names

**Developer Checklist**


**Testing Instructions**

1. Open an Open Response Assessment with file uploads enabled
2. Attempt to upload a file with non-ASCII characters in the name (e.g. `tęšt-fïłê.png`) 
3. Upload should succeed (previously, uploading file would fail with an error on the frontend)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:


FYI: @edx/masters-devs-gta
